tools/carveme/carveme.xml

```diff
@@ -0,0 +1,140 @@
+    </description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.6.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <xml name="biotools">
+            <xrefs>
+                <xref type="bio.tools">CarveMe</xref>
+            </xrefs>
+        </xml>
+    </macros>
+    <requirements>
+        <requirement type='package' version="@TOOL_VERSION@">carveme</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    
+    carve '$annotation'
+    #if $dna
+        --dna
+    #end if
+    #if $egg
+        --egg 
+    #end if
+    #if $fc2
+        --fc2 
+    #end if
+    #if $cobra
+        --cobra 
+    #end if
+    #if $diamond_args
+        --diamond-args '$diamond_args'
+    #end if
+    #if $universe
+        --universe '$universe'
+    #end if 
+    #if $universe_file
+        --universe_file '$universe_file'
+    #end if
+    #if $mediadb
+        --mediadb '$mediadb'
+    #end if
+    #if $soft
+        --soft '$soft'
+    #end if
+    #if $hard
+        --hard '$hard'
+    #end if
+    #if $reference
+        --reference '$reference'
+    #end if
+    --solver scip
+    --output "$model" 
+    
+    
+    ]]></command>
+    <inputs>
+    <param name="annotation" label="Annotation file" type="data" format="data" optional="false" multiple="false"/>
+    <param argument="--dna" label="Build from DNA fasta file" type="boolean" truevalue="--dna" falsevalue="" optional="true"/>
+    <param argument="--egg" label="Build from eggNOG-mapper output file"  type="boolean" truevalue="--egg" falsevalue="" optional="true"/>
+    <param argument="--fc2" label="Output SBML in sbml-fbc2 format" type="boolean" truevalue="--fc2" falsevalue="" optional="true"/>
+    <param argument="--cobra" label="Output SBML in old cobra format" type="boolean" truevalue="--cobra" falsevalue="" optional="true"/>
+    <param argument="--diamond-args" label="Additional arguments for running diamond"  type="text" optional="true"/>
+    <param argument="--universe" label="Pre-built universe model (default: bacteria)"  type="text" optional="true"/>
+    <param argument="--universe_file" label="Reaction universe file (SBML format)"  type="data" format="data" optional="true"/>
+    <param argument="--mediadb" label="Media database file"  type="data" format="data" optional="true"/>
+    <param argument="--soft" label="Soft constraints file"  type="data" format="data" optional="true"/>
+    <param argument="--hard" label="Hard constraints file"  type="data" format="data" optional="true"/>
+    <param argument="--reference" label="Manually curated model of a close reference species"  type="file" optional="true"/>
+    </inputs>
+    <outputs>
+        <data name="model" format="xml"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="annotation" value="protein.faa"/>
+            <output name="model" value="modeltest.xml" ftype='xml' />
+        </test>
+    </tests>
+    <help>
+
```

Comment:
This kind of help does not add much benefit to the user, can you maybe explain what the tool does and give some general hints. The parameters are already explained above with metadata, we don't need to add it here.